vector-stream/vector-stream.cabal

```diff
@@ -50,7 +50,7 @@ Library
         src
 
   Build-Depends: base >= 4.9 && < 4.22
-               , ghc-prim >= 0.2 && < 0.12
+               , ghc-prim >= 0.2 && < 0.14
```

Comment:
I wonder why ghc devs skipped `ghc-prim-0.12` version. Seems strange, but I can confirm that the [new version will be `ghc-prim-0.13`](https://gitlab.haskell.org/ghc/ghc/-/commit/6fc1fa3bdc8f53acdb19e47145789274060e498f#4c196af54dc163066e3b52830ae884ffb25798be)